### PR TITLE
Add In Decree Absolute Listeners

### PR DESCRIPTION
**Pull request by tchow8**
Enables listeners for Decree Absolute.

### CODE REVIEW

File: app-gateway.tf [hunk 1]
```diff
@@ -44,6 +51,11 @@ module "appGw" {
       data     = "${data.azurerm_key_vault_secret.dn_cert.value}"
       password = ""
     },
+      {
```
Comment (qzhou-hmcts) [inline app-gateway.tf]: wrong indentation

File: app-gateway.tf [hunk 2]
```diff
@@ -80,6 +92,22 @@ module "appGw" {
       SslCertificate          = "${var.dn_external_cert_name}${local.dn_suffix}"
       hostName                = "${var.dn_external_hostname}"
     },
+        {
```
Comment (qzhou-hmcts) [inline app-gateway.tf]: wrong indentation

File: app-gateway.tf (hunk 1 above)
Comment (tchow8) [reply app-gateway.tf]: fixed

File: app-gateway.tf (hunk 2 above)
Comment (tchow8) [reply app-gateway.tf]: fixed

**Pull request merged**